routes/errorStatusCodes.js

```diff
@@ -0,0 +1,102 @@
+var express = require('express');
+var router = express.Router();
+var utils = require('../util/utils')
+
+// valid Pet
+var tommyPet = {
+    "name":"Tommy Tomson",
+    "aniType":"Dog"
+};
+
+// unexpected enum
+var sadCasper = {
+    "actionResponse":"grrrr",
+    "errorType":"PetSadError",
+    "errorMessage":"casper aint happy",
+    "reason":"need more treats"
+};
+
+// enum from allowed values
+var hungryScooby = {
+    "actionResponse":"howl",
+    "errorType":"PetHungryOrThirstyError",
+    "errorMessage":"scooby is low",
+    "reason":"need more everything",
+    "hungryOrThirsty":"hungry and thirsty"
+};
+
+var stringError = " is missing";
+
+var intError = 123;
+
+var linkNotFoundError = {
+    "someBaseProp":"problem finding pet",
+    "reason":"link to pet not found",
+    "whatSubAddress":"pet/yourpet was not found",
+    "whatNotFound":"InvalidResourceLink"
+};
+
+var animalNotFoundError = {
+    "someBaseProp":"problem finding animal",
+    "reason":"the type of animal requested is not available",
+    "name":"coyote",
+    "whatNotFound":"AnimalNotFound"
+};
+
+
+var pathitem = function(coverage) {
+    router.post('/Pets/doSomething/:whatAction', function(req, res, next) {
+        var whatAction = req.params.whatAction;
+        console.log('Inside action: "' + whatAction +'"\n');
+        if (whatAction === 'stay') {
+            coverage['expectedNoErrors']++;
+            res.status(200).send(JSON.stringify(tommyPet));
```

Comment:
Please use "end" and not "send", this was the bug that was returning HTML content-type